### ISSUE TITLE
Webpage Almost Unreadable

### ISSUE THREAD

**Issue by KevinCooper** [jtpereyda/boofuzz#262]
https://github.com/jtpereyda/boofuzz/blob/81d86e1210baa00713265dc42727f79e5e6c8dc6/boofuzz/web/app.py#L147

In large stack traces, then the web page will dump the whole stack trace into the `Crash Synopsis` portion of the page.  This makes the webpage huge and hard to follow during test.  Pausing makes it possible to scroll to the bottom and click through the test cases.  Is this by design?

The text being output to page is being generate/added to the list at:
 https://github.com/jtpereyda/boofuzz/blob/7a977d7f45d41804a30a0be6cb76de60a5cf2595/boofuzz/sessions.py#L802

**Comment by jtpereyda** [comment]
@KevinCooper 
> Pausing makes it possible to scroll to the bottom and click through the test cases. Is this by design?

No, probably it just hasn't been well-tested in this case. What makes the page unreadable? Is it slow to scroll? Slow to respond to actions?

**Comment by SR4ven** [comment]
@jtpereyda I think the problem here is that long failure messages break the webpage layout as I pointed out here https://github.com/jtpereyda/boofuzz/pull/248#issuecomment-471191242 a while ago. They should be truncated before showing them in a GUI.

**Comment by jtpereyda** [comment]
@SR4ven There should be a reasonable solution to that issue... I'll look at it next time I get some time for maintenance.

@KevinCooper you mentioned
> Pausing makes it possible to scroll to the bottom and click through the test cases. 
You seem to be implying that scrolling to the bottom doesn't work when it's not paused, and only when there is a large stack trace -- is that correct?

**Comment by SR4ven** [comment]
Should have been fixed by #268 
If not it, I'm quite sure it will be fixed in #491
